core/jreleaser-model-impl/src/main/java/org/jreleaser/model/internal/extensions/Extension.java

```diff
@@ -65,6 +66,11 @@ public String getDirectory() {
             return directory;
         }
 
+        @Override
```

Comment:
Please update the serialVersionUID at 

https://github.com/jreleaser/jreleaser/blob/522d6f9fd73600f07cd1fa54798436ecdc60b287/core/jreleaser-model-impl/src/main/java/org/jreleaser/model/internal/extensions/Extension.java#L39
https://github.com/jreleaser/jreleaser/blob/522d6f9fd73600f07cd1fa54798436ecdc60b287/core/jreleaser-model-impl/src/main/java/org/jreleaser/model/internal/extensions/Extension.java#L49